Xamarin.Essentials/Connectivity/Connectivity.android.cs

```diff
@@ -21,6 +22,20 @@ static void StartListeners()
             Platform.AppContext.RegisterReceiver(conectivityReceiver, new IntentFilter(ConnectivityManager.ConnectivityAction));
         }
 
+        static SignalStrength PlatformSignalStrength()
+        {
+            var wifiManager = Platform.WifiManager;
+            if (wifiManager is null)
```

Comment:
This is redundant as you do wifiManager?. below so you can remove this

Reply:
Acutally I would rather remove the ?. as this check is necessary to return unknown value if we do not have a wifi Manager instance available if that were acceptable